### PR TITLE
PPO - Add hyperparam tuning with ray.tune

### PR DESCRIPTION
**Pull request by xeviknal**
PPO has way more hyperparams compared with REINFORCE ones.
- It is really easy to make the models overfit by doing many passes of the very same transitions
- Loss must be really watched closely in order to inspect its 3 components (and therefore its 2 coefficients)

This PR adds hyperparam tuning tooling to help us see what's the best configuration.

### CODE REVIEW

**Comment by xeviknal** [comment]
Up and running! Hope it gets the 10 experiments done :crossed_fingers: 
Ray-Dashboard

**Comment by xeviknal** [comment]
Screenshot of TensorBoard (5)

**Comment by xeviknal** [comment]
Results in yann lecun -> ~/ray_results/train_2021-04-14_10-01-45/
yanlecun-tune

**Comment by xeviknal** [comment]
legend-train_2021-04-14_10-01-45
tunning-train_2021-04-14_10-01-45
